### PR TITLE
Dont use JSON as templating medium

### PR DESCRIPTION
**Pull request by ivanvc**
Remove using JSON as the templating medium to generate Kubernetes Job payloads.

### CODE REVIEW

**Comment by tomcruz17** [comment]
👀 That's a lot. I'll continue reviewing this later/early tomorrow.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+name: Tests
+
+on: [push]
+
+jobs:
+  build:
+
```
Comment (tomcruz17) [inline .github/workflows/test.yml:7]: Why the new line here?

Comment (ivanvc) [reply .github/workflows/test.yml:7]: Copy pasta from GitHub's documentation

File: pkg/template/job_test.go [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						corev1.Container{
+							Name:    "pi",
+							Image:   "alpine:{{ uuidv4 }}",
+							Command: []string{"exit", "1"},
+							Env: []corev1.EnvVar{
+								corev1.EnvVar{
+									Name:  "PAYLOAD",
+									Value: `{{ if fromJson .Payload }}{{ .Payload | fromJson | toJson }}{{ else }}{{ .Payload }}{{ end }}`,
```
Comment (tomcruz17) [inline pkg/template/job_test.go:30]: So this confirms that `fromJson` without the need of `replace "\n" ""`. 👍🏽

**Review by tomcruz17 — APPROVED** [review]
Congrats on solving your leet code hard 👏🏽

**Pull request merged**